frontend/src/utils/events.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * Ignore events bubbling from children. Common use case would be
+ * ignoring @sl-hide events from tooltips within a dialog.
+ *
+ * @example Usage:
+ * ```
+ * <btrix-element
+ *   @sl-hide=${makeCurrentTargetHandler(this)(() => console.log("Only current target!"))}
+ * >
+ * </btrix-element>
+ * ```
+ */
+export function makeCurrentTargetHandler(t: EventTarget) {
```

Comment:
@emma-sg FYI I just pushed this after your review

Reply:
Nice, looks good!